src/lang/std/engineConnection.ts

```diff
@@ -1088,7 +1092,7 @@ export enum EngineCommandManagerEvents {
  * of those commands. It also sets up and tears down the connection to the Engine
  * through the {@link EngineConnection} class.
  *
- * It also maintains an {@link artifactMap} that keeps track of the state of each
+ * It also maintains an {@link artifactGraph} that keeps track of the state of each
```

Comment:
I don't believe this comment is true; the artifactGraph doesn't track the state of the commands, the `ResponseMap` does

Reply:
Updated.